README.md

```diff
@@ -20,8 +23,16 @@ Another use case for writing plugins is importing data from sources other than f
 ### How to write an import plugin?
 Getting started is as easy as creating a new C# project in your favorite IDE, adding a plugin manifest and linking the [Zeiss.PiWeb.Sdk.Import](https://www.nuget.org/packages/Zeiss.PiWeb.Sdk.Import/) NuGet. To make this even easier, we provide a project template you can use as a starting point. More detailed information about this can be found in our [import plugin documentation](https://zeiss-piweb.github.io/PiWeb-Import-Sdk/).
 
+### Version compatibilities
+The following PiWeb versions are compatible with the below import SDK versions:

```

Comment:
Maybe we can use the opportunity here to explain a bit more what we mean by compatibility.
```suggestion
The following table lists which versions of PiWeb will support plugins written against which version of the Import SDK.
```